camel-idea-plugin/src/main/java/com/github/cameltooling/idea/annotator/CamelBeanMethodAnnotator.java

```diff
@@ -112,26 +116,11 @@ public void annotate(@NotNull PsiElement element, @NotNull AnnotationHolder hold
 
     @NotNull
     public List<PsiMethod> getMatchingMethods(PsiClass psiClass, String methodName) {
-        return getJavaMethodUtils().getBeanMethods(getJavaMethodUtils().getMethods(psiClass))
+        final JavaMethodUtils javaMethodUtils = JavaMethodUtils.getService();
+        return javaMethodUtils.getBeanMethods(javaMethodUtils.getMethods(psiClass))
```

Comment:
Why adding a field instead of calling directly the method?

Reply:
It's used twice on line 120.